### PR TITLE
Implementation of RedeemIntentConfirmed Handler

### LINKED ISSUE #146: Redeem intent confirm handler
- Implement RedeemIntentDeclaredHandler
- Update targetStatus to Declared.

### PR DESCRIPTION
**Pull request by abhayks1**
PR covers below:

- Implementation of redeem intent confirmed handler
- Tests

Fixes #146

### CODE REVIEW

File: src/handlers/Handlers.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ import StakeProgressedHandler from './stake_and_mint/StakeProgressedHandler';
 import StakeRequestedHandler from './stake_and_mint/StakeRequestedHandler';
 import StakeIntentConfirmedHandler from './stake_and_mint/StakeIntentConfirmedHandler';
 import RedeemIntentDeclaredHandler from './redeem_and_unstake/RedeemIntentDeclaredHandler';
+import RedeemIntentConfirmedHandler from "./redeem_and_unstake/RedeemIntentConfirmedHandler";
```
Comment (puneet-khushwani-eth) [inline src/handlers/Handlers.ts]:
```suggestion
import RedeemIntentConfirmedHandler from './redeem_and_unstake/RedeemIntentConfirmedHandler';
```

File: src/handlers/redeem_and_unstake/RedeemIntentConfirmedHandler.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+          message.targetStatus = MessageStatus.Declared;
+        }
+        return message;
+      },
+    ));
+
+    const savePromises = [];
+    for (let i = 0; i < models.length; i += 1) {
+      Logger.debug(`Changing target status to declared for message hash ${models[i].messageHash}`);
+      savePromises.push(this.messageRepository.save(models[i]));
```
Comment (puneet-khushwani-eth) [inline src/handlers/redeem_and_unstake/RedeemIntentConfirmedHandler.ts]: similar to previous PR feedback, may be .catch on individual queries

File: test/handlers/redeem_and_unstake/RedeemIntentConfirmedHandler/persist.test.ts [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+      undefined,
+      undefined,
+    );
+
+    assert.equal(
+      models.length,
+      transactions.length,
+      'Number of models must be equal to transaction',
+    );
+
```
Comment (puneet-khushwani-eth) [inline test/handlers/redeem_and_unstake/RedeemIntentConfirmedHandler/persist.test.ts]: may be also add a test case for RevocationDeclared

Comment (abhayks1) [reply test/handlers/redeem_and_unstake/RedeemIntentConfirmedHandler/persist.test.ts]: Done!

File: src/handlers/Handlers.ts (hunk 1 above)
Comment (abhayks1) [reply src/handlers/Handlers.ts]: Done!

File: src/handlers/redeem_and_unstake/RedeemIntentConfirmedHandler.ts (hunk 2 above)
Comment (abhayks1) [reply src/handlers/redeem_and_unstake/RedeemIntentConfirmedHandler.ts]: Done!

**Comment by abhayks1** [comment]
Available for review 🐰

**Pull request merged**